### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: CONTRIBUTING [hunk 1]
```diff
@@ -40,7 +40,7 @@ imported into LXC).
 
 Language bindings for the libraries need to be released under LGPLv2.1+.
 
-Anything else (non-libaries) needs to be Free Software and needs to be
+Anything else (non-libraries) needs to be Free Software and needs to be
```
Comment (jsoref) [inline CONTRIBUTING]: It's possible this is a license file, in which case you probably don't want to change it, or you'd be better served by refreshing the file w/ the latest version

File: src/lxc/cmd/lxc_user_nic.c [hunk 2]
```diff
@@ -1319,7 +1319,7 @@ int main(int argc, char *argv[])
 		_exit(EXIT_FAILURE);
 	}
 
-	/* Write names of veth pairs and their ifindeces to stout:
+	/* Write names of veth pairs and their ifindices to stout:
```
Comment (jsoref) [inline src/lxc/cmd/lxc_user_nic.c]: This is the one change I'm least confident about.

File: hooks/dhclient-script [hunk 3]
```diff
@@ -32,7 +32,7 @@
 # Resolve every paths against the $ROOTFS environment variable.
 
 # 'ip' just looks too weird. Also, we now have unit-tests! Those unit-tests
-# overwirte this line to use a fake ip-echo tool. It's also convenient
+# override this line to use a fake ip-echo tool. It's also convenient
```
Comment (jsoref) [inline hooks/dhclient-script]: I'm not sure about this, it could be `overwrite` or `override`. I'm guessing the file isn't actually being rewritten.

**Comment by brauner** [comment]
You need to sign-off all of these commits. :)

**Comment by jsoref** [comment]
I'll sign off once someone is ok w/ the set. Also, do you want individual commits or a collapsed commit?

**Comment by stgraber** [comment]
@jsoref We can't legally look at any of your commits until they have the Signed-off-by line in the commit message so you'll need to update them first, then when the bot says DCO is okay, we can start looking at them.

I think individual commits make the most sense for this, easier to backport.

**Comment by jsoref** [comment]
@brauner: I updated this...

**Comment by brauner** [comment]
jenkins: test this please

**Comment by brauner** [comment]
Thanks!

**Pull request merged**